### PR TITLE
Follow-up edits to PR#2695

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/2695

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @bfallonf  @tpoitras PTAL. Thanks!

File: architecture/additional_concepts/admission_controllers.adoc [hunk 1]
```diff
@@ -86,23 +88,25 @@ admissionConfig:
...
 
-If you were using `admissionConfig` elements before that cannot be safely combined into a single
-admission chain, you will get a warning in your API server logs and your API server will start with
-two separate admission chains for legacy compatibility.  You should update your `admissionConfig` to resolve
-the warning. 
+If you were previously using `*admissionConfig*` elements that cannot be safely
+combined into a single admission chain, you will get a warning in your API
+server logs and your API server will start with two separate admission chains
+for legacy compatibility. Update your `*admissionConfig*` to resolve the
+warning.
```
Comment (bfallonf) [inline architecture/additional_concepts/admission_controllers.adoc]: Should the above be in a notebox?

**Comment by bfallonf** [comment]
@ahardin-rh One suggestion from me, but otherwise LG.

**Comment by ahardin-rh** [comment]
Fixed. Thanks @bfallonf

**Comment by ahardin-rh** [comment]
Rev history in original PR

**Pull request merged**